samples/secure-with-tls/docker-compose.certs.yml

```diff
@@ -16,7 +16,7 @@ services:
     network_mode: none
   
   cert-gen:
-    image: eventstore/es-gencert-cli:1.3.0
+    image: eventstore/es-gencert-cli:latest.0
```

Comment:
```suggestion
    image: docker.eventstore.com/eventstore-utils/es-gencert-cli:latest
```

I think you missed this one. Although it's just samples :)

Reply:
Ty, good catch